### DISCUSSION TITLE
Can't uninstall explorer patcher at all

### DISCUSSION THREAD

**Discussion by 02tz** [valinet/ExplorerPatcher#2645] [Q&A]
Renaming the installer to ep_uninstall will result in an error servicing explorer, running it from the command line results in the same error, uninstalling it from the settings app and no change, and the patcher is still installed on the machine. I wanted to reinstall since the volume button on the taskbar disappeared. ???

**Comment by pyrates999** [comment]
enable file extensions and make sure it is renamed to ep_uninstall.exe: https://github.com/valinet/ExplorerPatcher/wiki/Installer-How-To

And please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

Please include any other software you have installed that modifies the UI of windows.
